pages/index.tsx

```diff
@@ -88,30 +111,56 @@ export default function Home() {
       )
 
       const tx = new Transaction()
+      console.log(tx)
 
       if (additionalIxs?.length) {
         tx.add(...additionalIxs)
       }
 
       tx.add(...instructions)
 
-      tx.recentBlockhash = (await connection.getLatestBlockhash()).blockhash
+      tx.recentBlockhash = (await connection.getLatestBlockhash()).blockhash;
 
-      const txid = await wallet.sendTransaction(tx, connection, {
+    
+    var txid:string;
+    
+    try{
+      txid = await wallet.sendTransaction(tx, connection, {
         signers: [mint],
-      })
-
-      const latest = await connection.getLatestBlockhash()
-      await connection.confirmTransaction({
-        blockhash: latest.blockhash,
-        lastValidBlockHeight: latest.lastValidBlockHeight,
-        signature: txid,
-      })
+      })    
+    }
+    catch(e){
+      setFormMessage('mint failed');
+      return
+    }
+
+    while (true){
```

Comment:
`while (true)` is not an appropriate way of reaching the desired result, it would end up with too many loops. Try using `setInterval`, with a 1000ms interval between the calls.